app/models/classification_pipeline.rb

```diff
@@ -52,50 +54,47 @@ def extract(classification)
     raise
   end
 
-  def reduce(workflow_id, subject_id, user_id)
+  def reduce(workflow_id, subject_id, user_id, extract_ids=[])
```

Comment:
This method is getting really long. Should it get broken up into various things that have less concerns? Maybe to start with, split it up into one that does subjecty-things, one that does usery-things, and then both call a third method with commonalities. Perhaps also move it to its own class(es)?

Reply:
I think that my most recent refactor considerably shortens it, but if we need to split it up i'd think maybe having "compute reductions" and "store reductions" methods would be the way to go

Reply:
No, it's a lot smaller now so consider this resolved.